compose/project.py

```diff
@@ -60,13 +60,15 @@ class Project(object):
     """
     A collection of services.
     """
-    def __init__(self, name, services, client, networks=None, volumes=None, config_version=None):
+    def __init__(self, name, services, client, networks=None, volumes=None, config_version=None,
+                 noansi=False):
```

Comment:
I wish there was a better way to pass this value, as it strictly influences CLI / output and hence doesn't really belong on the `Project` class. Unfortunately this would probably require a code overhaul that's outside of the scope of this PR.